# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/system/NetworkGeolocationProvider.jsm
# Commit: cefcb5f4f4e0
# Full Hash: cefcb5f4f4e037cc90d4d87e474020ddac411a7b
# Author: David Parks <daparks@mozilla.com>
# Date: 2023-05-13 21:42:25
# Regressor Bug: 1810421
# File Overlap Count: 2
# Description:
#   Bug 1810421: Only scan wifi when the network changes or when on mobile r=necko-reviewers,kershaw a=reland CLOSED TREE
#   
#   Simplifies the concurrent operations of wifi scanning and reduces the
#   frequency in common cases.  Wifi scanning when on mobile is reduced from
#   every 5 seconds to every minute.  Wifi scans will also happen whenever
# ==============================================================================

diff -r af2b512cb5ac -r cefcb5f4f4e0 dom/system/NetworkGeolocationProvider.jsm
--- a/dom/system/NetworkGeolocationProvider.jsm	Sat May 13 00:36:00 2023 +0000
+++ b/dom/system/NetworkGeolocationProvider.jsm	Sat May 13 00:36:01 2023 +0000
@@ -318,7 +318,7 @@
       this.wifiService = Cc["@mozilla.org/wifi/monitor;1"].getService(
         Ci.nsIWifiMonitor
       );
-      this.wifiService.startWatching(this);
+      this.wifiService.startWatching(this, false);
     }
 
     this.resetTimer();